benchmark/wh_bench.c

```
@@ -459,7 +509,7 @@ static int _runClientBenchmarks(whClientContext* client)
 
 /* Initializes a client context based on the provided config, runs the
  * benchmarks, then cleans up the context */
-int wh_Bench_ClientCfg(whClientConfig* clientCfg)
+int wh_Bench_ClientCfg(whClientConfig* clientCfg, int transport)
```

We should check the transport against the enabled build options - for example if you pass the POSIX transport enum on a non-POSIX system (e.g. bare metal) then this should throw an error.